tests/manifests/test_pod_idempotency.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 New Vector Ltd
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+import random
+
+import pytest
+import yaml
+
+from . import values_files_to_test
+from .utils import helm_template, template_id
+
+
+@pytest.mark.parametrize("values_file", ["element-web-minimal-values.yaml"])
+@pytest.mark.asyncio_cooperative
+async def test_values_file_renders_idempotent_pods(release_name, values, helm_client, temp_chart):
+    async def _patch_version_chart():
+        with open(f"{temp_chart}/Chart.yaml") as f:
+            chart = yaml.safe_load(f)
+        with open(f"{temp_chart}/Chart.yaml", "w") as f:
+            chart["version"] = f"{random.randint(0, 100)}.{random.randint(0, 100)}.0"
```

Comment:
There's a chance this could match what's already there. Add a suffix or interpret the chart version and explicitly increment?